cli/gitcommands.go

```diff
@@ -1,36 +1,93 @@
 package cli
 
 import (
+	"os"
+	"strings"
+
 	"github.com/jfrog/froggit-go/vcsutils"
+	outputFormat "github.com/jfrog/jfrog-cli-core/v2/common/format"
 	"github.com/jfrog/jfrog-cli-core/v2/common/progressbar"
+	pluginsCommon "github.com/jfrog/jfrog-cli-core/v2/plugins/common"
 	"github.com/jfrog/jfrog-cli-core/v2/plugins/components"
 	flags "github.com/jfrog/jfrog-cli-security/cli/docs"
-	gitDocs "github.com/jfrog/jfrog-cli-security/cli/docs/git"
-	"github.com/jfrog/jfrog-cli-security/commands/git"
+	gitAuditDocs "github.com/jfrog/jfrog-cli-security/cli/docs/git/audit"
+	gitContributorsDocs "github.com/jfrog/jfrog-cli-security/cli/docs/git/contributors"
+	"github.com/jfrog/jfrog-cli-security/commands/git/audit"
+	"github.com/jfrog/jfrog-cli-security/commands/git/contributors"
+	"github.com/jfrog/jfrog-cli-security/utils/xsc"
 	"github.com/jfrog/jfrog-client-go/utils/errorutils"
-	"os"
-	"strings"
 )
 
 func getGitNameSpaceCommands() []components.Command {
 	return []components.Command{
+		{
+			Name:        "audit",
+			Aliases:     []string{"a"},
+			Description: gitAuditDocs.GetDescription(),
+			Flags:       flags.GetCommandFlags(flags.GitAudit),
+			Hidden:      true,
+			Action:      GitAuditCmd,
+		},
+		// TODO: Move cc cmd to Frogbot/Script
 		{
 			Name:        "count-contributors",
 			Aliases:     []string{"cc"},
 			Flags:       flags.GetCommandFlags(flags.GitCountContributors),
-			Description: gitDocs.GetContContributorsDescription(),
+			Description: gitContributorsDocs.GetContContributorsDescription(),
 			Hidden:      true,
 			Action:      GitCountContributorsCmd,
 		},
 	}
 }
 
-func GetCountContributorsParams(c *components.Context) (*git.CountContributorsParams, error) {
-	params := git.CountContributorsParams{}
+func GitAuditCmd(c *components.Context) error {
+	gitAuditCmd := audit.NewGitAuditCommand()
+	// Set connection params
+	serverDetails, err := createServerDetailsWithConfigOffer(c)
+	if err != nil {
+		return err
+	}
+	xrayVersion, xscVersion, err := xsc.GetJfrogServicesVersion(serverDetails)
+	if err != nil {
+		return err
+	}
+	gitAuditCmd.SetServerDetails(serverDetails).SetXrayVersion(xrayVersion).SetXscVersion(xscVersion)
+	// Set violations params
+	if err = validateXrayContext(c, serverDetails); err != nil {
```

Comment:
I think we should change the name of this function. Maybe validateXrayViolationContext?